DNN Platform/Dnn.AuthServices.Jwt/Data/Scripts/01.00.00.SqlDataProvider

```diff
@@ -0,0 +1,172 @@
+/************************************************************/
+/*****              SqlDataProvider                     *****/
+/*****                                                  *****/
+/*****                                                  *****/
+/***** Note: To manually execute this script you must   *****/
+/*****       perform a search and replace operation     *****/
+/*****       for {databaseOwner} and {objectQualifier}  *****/
+/*****                                                  *****/
+/************************************************************/
+
+/************************************************************
+ ***** TABLE: JsonWebTokens
+ ************************************************************/
+IF OBJECT_ID(N'{databaseOwner}{objectQualifier}JsonWebTokens', N'U') IS NULL
+BEGIN
+	CREATE TABLE {databaseOwner}[{objectQualifier}JsonWebTokens]
+	(
+		[TokenId]           VARCHAR(36) NOT NULL,   -- this is not a unicode string!
+		[UserId]            INT NOT NULL,
+		[CreatedOn]         SMALLDATETIME DEFAULT (GETUTCDATE()),
+		[RenewCount]        INT DEFAULT (0),
+		[TokenExpiry]       SMALLDATETIME NOT NULL, -- access token expiry UTC timestamp
+		[RenewalExpiry]     SMALLDATETIME NOT NULL, -- renewal token expiry UTC timestamp
+		[TokenHash]         VARCHAR(96) NOT NULL,   -- access token SHA1 (for validation)
+		[RenewalHash]       VARCHAR(96) NOT NULL,   -- renewal token SHA1 (for validation)
+		CONSTRAINT [PK_{objectQualifier}JsonWebTokens] PRIMARY KEY ([TokenId])
+	)
+
+	CREATE NONCLUSTERED INDEX [IX_{objectQualifier}JsonWebTokens_UsertId]
+		ON {databaseOwner}[{objectQualifier}JsonWebTokens] ([UserId])
+
+	CREATE NONCLUSTERED INDEX [IX_{objectQualifier}JsonWebTokens_RenewalExpiry]
+		ON {databaseOwner}[{objectQualifier}JsonWebTokens] ([RenewalExpiry])
+END
+GO
+
+/************************************************************
+ ***** SPROC: JsonWebTokens_GetById
+ ************************************************************/
+IF OBJECT_ID(N'{databaseOwner}[{objectQualifier}JsonWebTokens_GetById]', N'P') IS NULL
+	EXEC('CREATE PROCEDURE {databaseOwner}[{objectQualifier}JsonWebTokens_GetById] AS BEGIN SELECT 1 END');
+GO
+
+ALTER PROCEDURE {databaseOwner}[{objectQualifier}JsonWebTokens_GetById]
+	@TokenId VARCHAR(36)
+AS
+BEGIN
+	SELECT TOP(1) *
```

Comment:
unless there's some significant performance benefit, my opinion is that the readability of this procedure is hindered by including `TOP(1)` when we're querying by the primary key (i.e. it makes the reader question whether there would be cases where multiple rows would match the criteria).

Also, I'd prefer the full column list, rather than `*`